cellxgene_schema_cli/scripts/gene_processing.py

```diff
@@ -30,59 +34,103 @@ def _parse_gtf(gtf_path: str, output_file: str):
             # Desired features based on whether is gene or transcript
             if line[2] == "gene":
                 features = ["gene_id", "gene_name", "gene_version"]
-            elif line[2] == "transcript":
-                features = ["transcript_id", "transcript_name", "transcript_version"]
+            # TRANSCRIPT implementation
+            #elif line[2] == "transcript":
```

Comment:
Remove

Reply:
Also do you need to update the doc string for the function to remove references to transcripts?